mk/rabbitmq-dist.mk

```diff
@@ -123,23 +163,34 @@ $(DIST_DIR)/%.ez:
 		&& grep -q '^prepare-dist::' $(SRC_DIR)/Makefile) || \
 		$(MAKE) --no-print-directory -C $(SRC_DIR) prepare-dist \
 		APP=$(APP) VSN=$(VSN) EZ_DIR=$(EZ_DIR)
-	$(verbose) (cd $(DIST_DIR) && $(ZIP) $(ZIP_V) -r $*.ez $*)
+	$(verbose) (cd $(DIST_DIR) && $(ZIP) $(ZIP_V) -r `basename $(EZ)` `basename $(EZ) .ez`)
 	$(verbose) rm -rf $(EZ_DIR)
 
+$(MIX_DIST_EZS): $(mix_task_archive_deps_installed)
+	$(verbose) rm -rf $(DIST_DIR)/tmp
+	$(verbose) mkdir $(DIST_DIR)/tmp
+	$(verbose) cd $(SRC_DIR) && MIX do deps.get, deps.compile, compile, archive.build.all -e -o $(realpath $(DIST_DIR))/tmp
```

Comment:
You can use `$(dist_verbose)` for this line so that the build is reported in the default case.